iotile_ext_cloud/iotile/cloud/cloud.py

```diff
@@ -22,14 +24,25 @@ def __init__(self):
 
         domain = conf.get('cloud:server')
 
+        self.api = Api(domain=domain)
+
         try:
             token = reg.get_config('arch:cloud_token')
+            self.api.set_token(token)
         except ArgumentError:
-            raise ExternalError("No stored iotile cloud authentication information", suggestion='Call iotile config link_cloud with your iotile cloud username and password')
+            # If we are interactive, try to get the user to login for a single
+            # session rather than making them call link_cloud to store a cloud token
+            if type_system.interactive:
+                username = raw_input("Please enter your iotile.cloud username: ")
```

Comment:
Any way we can pass the email via an argument too? So we can support the `-u` type option?